Tests/Components Tests/IssuerList/IssuerListComponentTests.swift

```diff
@@ -54,6 +61,29 @@ class IssuerListComponentTests: XCTestCase {
 
         waitForExpectations(timeout: 10)
     }
+    
+    func testSearch() {
+        // Given
+        let analyticsProviderMock = AnalyticsProviderMock()
+        let context = Dummy.context(with: analyticsProviderMock)
+        
+        sut = IssuerListComponent(paymentMethod: paymentMethod, context: context)
+        
+        let searchViewController = sut.viewController as! SearchViewController
+        let listViewController = searchViewController.resultsListViewController
+        
+        setupRootViewController(searchViewController)
+        
+        searchViewController.searchBar(searchViewController.searchBar, textDidChange: "3")
+        XCTAssertFalse(listViewController.view.isHidden)
+        XCTAssertEqual(analyticsProviderMock.infoCount, 1)
+        
+        searchViewController.searchBar(searchViewController.searchBar, textDidChange: "34")
+        wait(for: .seconds(1))
```

Comment:
We can add one more check _before_ this delay to ensure there is still one event sent. Otherwise even if you remove the Throttler, this test still would be green.

Small tip: always make your test red by changing the business logic (comment out the Throttler usage here) before commit

Reply:
Great tip, done!